pkg/gds/emails/client.go

```diff
@@ -366,6 +371,91 @@ func (m *EmailManager) SendExpiresAdminNotification(vasp *pb.VASP, reissueDate t
 	return 1, nil
 }
 
+// Helper function for HandleCertificateReissuance that sends reissuance reminder emails
+// a vasp's contacts, ensuring at least one of the contact's receives the reminder or a
+// critical alert is raised.
+func (m *EmailManager) SendContactReissuanceReminder(vasp *pb.VASP, timeWindow int, reissuanceDate time.Time) {
+	var (
+		verifiedContacts int
+		emailCount       int
+		serviceEmail     *mail.Address
+		err              error
+	)
+
+	ReissuanceData := ReissuanceReminderData{
+		VID:                 vasp.Id,
+		CommonName:          vasp.CommonName,
+		Endpoint:            vasp.TrisaEndpoint,
+		RegisteredDirectory: m.conf.ServiceEmail,
+		Reissuance:          reissuanceDate,
+	}
+
+	if vasp.IdentityCertificate != nil {
+		ReissuanceData.SerialNumber = strings.ToUpper(hex.EncodeToString(vasp.IdentityCertificate.SerialNumber))
+		ReissuanceData.Expiration, _ = time.Parse(time.RFC3339, vasp.IdentityCertificate.NotAfter)
+	}
+
+	if serviceEmail, err = mail.ParseAddress(m.serviceEmail.Address); err != nil {
+		log.Error().Err(err).Msg("could not parse the configured service email address")
+		return
+	}
```

Comment:
Maybe I'm missing something but don't we already have the `*mail.Address` on the certman struct?

Reply:
you're right, this is an artifact from when this was in a different package. Fixed!